library/src/main/java/com/instacart/library/truetime/TrueTime.java

```diff
@@ -58,11 +58,27 @@ public void initialize() throws IOException {
      * Cache TrueTime initialization information in SharedPreferences
      * This can help avoid additional TrueTime initialization on app kills
      */
-    public synchronized TrueTime withSharedPreferences(Context context) {
-        DISK_CACHE_CLIENT.enableDiskCaching(context);
+    public synchronized TrueTime withSharedPreferencesCache(Context context) {
+        DISK_CACHE_CLIENT.enableSharedPreferenceCaching(context);
```

Comment:
i'm wondering if we can shoehorn this to use the newer interface you created, so `DISK_CACHE_CLIENT.enableCacheInterface(sharedPrefCacheImpl);` ?

Reply:
Nice Suggestion. Will be updated.